oe_contact_forms.module

```diff
@@ -60,6 +60,9 @@ function oe_contact_forms_form_contact_form_form_alter(&$form, FormStateInterfac
   ];
 
   if ($is_corporate_form) {
+    // Remove disable message as it's not used in corporate contact forms.
```

Comment:
The problem here is that the first time you load the form the field will show and even if you mark the form as corporate it will still show until you save and reload the form.